docs/orchestrating-elastic-stack-applications/security/users-and-roles.asciidoc

```diff
@@ -23,6 +23,8 @@ For example, the password of the `elastic` user for an Elasticsearch cluster nam
 kubectl get secret quickstart-es-elastic-user -o go-template='{{.data.elastic | base64decode}}'
 ----
 
+To rotate this password, refer to: link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-rotate-credentials.html#k8s-rotate-credentials[Rotate auto-generated credentials].
```

Comment:
Let's use a internal cross reference:
```suggestion
To rotate this password, refer to <<{p}-rotate-credentials>>.
```

Reply:
OK, changed to an internal cross reference